src/shared/data/devices/devices.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { MobileIcon, TabletIcon, LaptopIcon, DesktopIcon, CheckmarkIcon } from '@nomios/web-uikit';
+
+export default [
+    {
+        id: 'mobile',
+        label: 'Mobile',
+        badge: <CheckmarkIcon />,
```

Comment:
As I said earlier, it feels strange to have the `badge` defined here. The rest of the fields is ok.

Reply:
To be fair, since this is only used in the device shared step, and the badge icon is the same for every device, we can completely remove it from here and directly apply it in the step code.

Reply:
Let’s do it